workspaces/entity-feedback/plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -191,6 +205,29 @@ export async function createRouter(
   router.post('/responses/:entityRef(*)', async (req, res) => {
     const { response, comments, consent } = req.body;
     const credentials = await httpAuth.credentials(req, { allow: ['user'] });
+    const { token } = await auth.getPluginRequestToken({
+      onBehalfOf: credentials,
+      targetPluginId: 'catalog',
+    });
+    const entityOwner =
+      (
+        await catalogClient.getEntityByRef(req.params.entityRef, {
+          token: token,
+        })
+      )?.relations?.find(rel => rel.type === 'ownedBy')?.targetRef || '';
```

Comment:
It's probably more robust to import and use this symbol from `@backstage/catalog-model` instead of hardcoding this here: 
https://github.com/backstage/backstage/blob/6b15e7c0183e819ab70ba07016b084f382fc970d/packages/catalog-model/src/kinds/relations.ts#L40